modules/argo-cd/templates/values.yaml

```diff
@@ -501,7 +501,14 @@ configs:
   # -- Repositories list to be used by applications
   ## Creates a secret for each key/value specified below to create repositories
   ## Note: the last example in the list would use a repository credential template, configured under "configs.credentialTemplates".
-  repositories: {}
+  repositories:
+    docker-registry:
+      url: registry-1.docker.io
+      # username: "docker"
+      # password: ""
+      name: docker-registry
+      enableOCI: "true"
+      type: "helm"
```

Comment:
When I was testing some helm charts that use the `oci://` prefix, this was needed.